setup.py

```diff
@@ -329,6 +335,7 @@ def parse_args():
     other = parser.add_argument_group('other', 'Miscellaneous arguments.')
     other.add_argument('--hdf5', help='Path to HDF5 root directory.')
     other.add_argument('--moab', help='Path to MOAB root directory.')
```

Comment:
Did we find out if this `--moab` specification still needed during install? I was hoping we could reduce this to only needing to specify `--dagmc`.

If we have pymoab, we have MOAB. If we need DAGMC, we better have MOAB too. The remaining question is maybe whether PyMOAB and DAGMC can rely on different installations of MOAB. They should be able to, as PyMOAB uses an runtime path setting in the dynamic libs and I think DAGMC does the same now as well.

Reply:
I've left this because some may want MOAB and not DAGMC

I've expanded on it in #54